docs/api/responses/meta.md

```diff
@@ -122,9 +124,20 @@ Used as a response for [`defineCatalogHandler`](../requests/defineCatalogHandler
 
 ``posterShape`` - _optional_ - string, can be `square` (1:1 aspect) or `regular` (1:0.675) or `landscape` (1:1.77). If you don't pass this, `regular` is assumed
 
-``background`` - _optional_ - string, the background shown on the stremio detail page ; heavily encouraged if you want your content to look good; URL to PNG, max file size 500kb
+#### Additional Parameters that are used for the Discover Page Sidebar:
 
-``logo`` - _optional_ - string, the logo shown on the stremio detail page ; encouraged if you want your content to look good; URL to PNG
+``background`` - _optional_ - string, the background shown on the stremio discover page in the sidebar; URL to PNG, max file size 500kb
```

Comment:
By default the discover sidebar seems to show a cropped version of the `poster`. This might be worth mentioning here.

Reply:
You're right, I removed `meta.background` from the Meta Preview docs, I also added a deprecation warning to `meta.trailers`, which will be an array of Strem Objects in the future.